.github/workflows/main.yml

```diff
@@ -87,8 +86,8 @@ jobs:
         id: docker_build
         uses: docker/build-push-action@v2
         with:
-          context: .
-          push: true
+          platforms: linux/amd64,linux/arm64/v7,linux/arm64/v8
+          push: ${{ github.event_name != 'pull_request' }}
```

Comment:
Same for here, I think `startsWith(github.ref, 'refs/tags/v')` is more secure.

Reply:
updated